test/service/test_jobs.py

```diff
@@ -184,7 +184,7 @@ def test_requests_tide_prediction(self):
         }, self.mock_requests.request_history[0].json())
 
     def test_fetches_correct_scene(self):
-        self.mock_get_algo.return_value = create_algorithm()
+        self.mock_get_algo.return_value = create_algorithm('pzsvc-ndwi-py')
```

Comment:
`pzsvc-ndwi-py` is the default value for the `create_algorithm` function.  You don't need to repeat it here.